src/support/azext_support/aaz/latest/support/in_subscription/tickets/_create.py

```diff
@@ -27,49 +27,49 @@ class Create(AAZCommand):
 
     :example: Create a ticket for Generic Quota increase for any Azure Service
-        az support in-subscription tickets create --contact-country "USA" --contact-email "abc@contoso.com" --contact-first-name "Foo" --contact-language "en-us" --contact-last-name "Bar" --contact-method "email" --contact-timezone "Pacific Standard Time" --description "QuotaTicketDescription" --diagnostic-consent "Yes" --problem-classification "/providers/Microsoft.Support/services/QuotaServiceNameGuid/problemClassifications/GenericProblemClassificationNameGuid" --severity "minimal" --ticket-name "GenericQuotaIncreaseRequest" --title "Request for Generic Quota Increase"
+        az support in-subscription tickets create --contact-country "USA" --contact-email "abc@contoso.com" --contact-first-name "Foo" --contact-language "en-us" --contact-last-name "Bar" --contact-method "email" --contact-timezone "Pacific Standard Time" --description "QuotaTicketDescription" --advanced-diagnostic-consent "Yes" --problem-classification "/providers/Microsoft.Support/services/QuotaServiceNameGuid/problemClassifications/GenericProblemClassificationNameGuid" --severity "minimal" --ticket-name "GenericQuotaIncreaseRequest" --title "Request for Generic Quota Increase"
 
     :example: Create a ticket to request an increase in Pools for a Batch account
-        az support in-subscription tickets create --contact-country "USA" --contact-email "abc@contoso.com" --contact-first-name "Foo" --contact-language "en-us" --contact-last-name "Bar" --contact-method "email" --contact-timezone "Pacific Standard Time" --description "QuotaTicketDescription" --diagnostic-consent "Yes" --problem-classification "/providers/Microsoft.Support/services/QuotaServiceNameGuid/problemClassifications/BatchQuotaProblemClassificationNameGuid" --severity "minimal" --ticket-name "BatchPoolsQuotaIncreaseRequest" --title "Request for Pools Quota Increase in Batch Account" --quota-change-version "1.0" --quota-change-requests [0].region="WESTUS" --quota-change-requests [0].payload="'{`"AccountName`": `"test`", `"NewLimit`": 102, `"Type`": `"Pools`"}'"
+        az support in-subscription tickets create --contact-country "USA" --contact-email "abc@contoso.com" --contact-first-name "Foo" --contact-language "en-us" --contact-last-name "Bar" --contact-method "email" --contact-timezone "Pacific Standard Time" --description "QuotaTicketDescription" --advanced-diagnostic-consent  "Yes" --problem-classification  "/providers/Microsoft.Support/services/quota_service_guid/problemClassifications/batch_problemClassification_guid" --severity "minimal" --ticket-name "BatchPoolsQuotaIncreaseRequest" --title "Request for Pools Quota Increase in Batch Account" --quota-change-version "1.0" --quota-change-requests [0].region="WESTUS" --quota-change-requests [0].payload="{AccountName: test, NewLimit: 102, Type: Pools}" --quota-change-subtype "Account"

```

Comment:
"{AccountName: test, NewLimit: 102, Type: Pools}" is space okay? Yesterday we had noticed it was complaing about space between characters

Reply:
<img width="950" alt="image" src="https://github.com/RudraSharma93Microsoft/azure-cli-extensions/assets/126628038/127f7f50-0aae-4b06-bb69-a4b17d9f019d">
No it's fine